src/components/CommentList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchCommentsbyArticleId } from "../utils/api";
+import CommentCard from "./CommentCard";
+
+const CommentList=()=> {
+    const[isLoading, setIsLoading] =useState(true);
+    const[commentsList, setCommentsList]= useState([]);
+    const params = useParams();
+    const articleId = params.article_id;
+
+    useEffect(() => {
+        setIsLoading(true);
+        fetchCommentsbyArticleId(articleId).then((comments) => {
+          setCommentsList(comments);
+          console.log(comments);
+        setIsLoading(false);
+        });
+      }, []);
+    
+    return (
+      <div>
```

Comment:
Could you use the loading state to give some feedback to the user that the comment are loading?